src/DataBuilder.php

```diff
@@ -775,10 +775,10 @@ public function scrub(&$data, $replacement = '*')
                 $parsedValues = array_values($parsed);
                 
                 /**
-                 * If we have at least one key/value pair (i.e. a=b) then
+                 * If we have at least two pairs (i.e. a=b&c=d) then
```

Comment:
If it is part of a URL then it will be found above in the first time we try to identify a URL with a query string. This is just if it is a bare string. I'll change it to exactly one `=` or if multiple `=` then also has more than one `&` which should be good enough.